### PR TITLE
Feature/grants 1608

### PR DESCRIPTION
**Pull request by zavies97**

### CODE REVIEW

**Review by salfedev — DISMISSED** [review]

File: app/config/question-bank-redux.js [hunk 1]
```diff
@@ -830,7 +830,7 @@ const questionBank = {
             heading: 'Eligibility',
             content: [
               {
-                para: `Your project cannot increase water use from summer abstraction or mains.\n\n
+                para: `Your project cannot increase water use from summer water surface abstraction or mains.\n\n
```
Comment (diya2214) [inline app/config/question-bank-redux.js:833]: revert here too

File: app/helpers/handlers.js [hunk 2]
```diff
@@ -182,7 +182,7 @@ const showPostPage = (currentQuestion, request, h) => {
     setYarValue(request, yarKey, dataObject)
   }
 
-  if (title === 'Will you {{_currentlyIrrigating_}} summer abstraction or mains?') {
+  if (title === 'Will you {{_currentlyIrrigating_}} summer water surface abstraction or mains?') {
     currentQuestion = {
```
Comment (diya2214) [inline app/helpers/handlers.js:186]: @zavies97  revert here as well

File: app/helpers/water-source-data.js [hunk 3]
```diff
@@ -3,8 +3,8 @@ const SUSTAINABLE_WATER_SOURCE = ['Winter peak-flow abstraction', 'Rain water ha
 const SUMMER_ABSTRACTION_MAINS_YES = 'increase your use of'
 const SUMMER_ABSTRACTION_MAINS_NO = 'use water from'
 const UNSUSTAINABLE_WATER_SOURCE = ['Summer water surface abstraction', 'Mains']
-const SUMMER_ABSTRACTION_MAINS_YES_ERROR = 'Select yes if you’re increasing use of water from summer abstraction or mains'
-const SUMMER_ABSTRACTION_MAINS_NO_ERROR = 'Select yes if you’re going to use summer abstraction or mains'
+const SUMMER_ABSTRACTION_MAINS_YES_ERROR = 'Select yes if you’re increasing use of water from summer water surface abstraction or mains'
+const SUMMER_ABSTRACTION_MAINS_NO_ERROR = 'Select yes if you’re going to use summer water surface abstraction or mains'
```
Comment (diya2214) [inline app/helpers/water-source-data.js:8]: revert these too

File: app/templates/change-summer-abstraction.njk [hunk 4]
```diff
@@ -46,7 +46,7 @@
       </form>
     </div>
     
-    {{ sideBar("Eligibility", "Your project cannot increase water use from summer abstraction or mains. \n\n RPA wants to fund projects that use more sustainable water sources such as:",
+    {{ sideBar("Eligibility", "Your project cannot increase water use from summer water surface abstraction or mains. \n\n RPA wants to fund projects that use more sustainable water sources such as:",
```
Comment (diya2214) [inline app/templates/change-summer-abstraction.njk]: revert please

File: app/config/question-bank-redux.js (hunk 1 above)
Comment (zavies97) [reply app/config/question-bank-redux.js]: should now be sorted

File: app/config/question-bank-redux.js [hunk 5]
```diff
@@ -792,7 +792,7 @@ const questionBank = {
       type: 'single-answer',
       minAnswerCount: 1,
       ineligibleContent: {
-        messageContent: `Your project cannot increase water use from summer abstraction or mains. 
+        messageContent: `Your project cannot increase water use from summer water surface abstraction or mains.
```
[the remark this answers is not in the document]
Comment (zavies97) [reply app/config/question-bank-redux.js]: should now be sorted

**Review by diya2214 — APPROVED** [review]

**Review by diya2214 — APPROVED** [review]

**Pull request merged**